src/main/recorder.ts

```diff
@@ -192,58 +241,53 @@ type RecorderOptionsType = {
             const isRaid = metadata.category == VideoCategory.Raids;
             const isLongEnough = (metadata.duration - overrun) >= this._options.minEncounterDuration;
 
-            if ((!isRaid || isLongEnough) && !discardVideo) {
-                // Cut the video to length and write its metadata JSON file.
-                // Await for this to finish before we return to waiting state.
-                await this.finalizeVideo(metadata, outputFilename);
-            } else {
+            if (isRaid && !isLongEnough) {
                 console.info("[Recorder] Raid encounter was too short, discarding");
+            } else if (discardVideo) {
```

Comment:
Does this discardVideo plumbing from `endRecording()` has a purpose? Can't see it in the code. If not, maybe should rip it out?

Reply:
I guess it doesn't anymore - it used to, but I can't find any references that uses it anymore.